Pluralize.xcodeproj/project.pbxproj

```diff
@@ -357,7 +357,8 @@
 				SDKROOT = iphoneos;
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_VERSION = 3.0;
+				SWIFT_SWIFT3_OBJC_INFERENCE = On;
+				SWIFT_VERSION = 4.2;
```

Comment:
The PR title says it's for Swift 5 - does "4.2" actually refer to Swift 5, or is the PR not up to date? 
Thanks for doing this, I really hope it gets merged.

Reply:
Hm.  This pull request was just running through the automatic update that Xcode guides you through.  I had this running with a project using Swift 5.

Since I don't think this PR will get merged, I did end up creating my own framework and posting it to cocoa pods. 

https://github.com/imattice/plurals

Reply:
Oh nice - I'll give your framework a try then!